src/test/resources/expected/test-attribute-2.kore.golden

```diff
@@ -1,79 +1 @@
-                    { sentenceSymbolSymbol =
-                        Symbol
-                            { symbolConstructor = Id Object "plus"
-                            , symbolParams = []
-                            }
-                    , sentenceSymbolSorts =
-                        [ SortActualSortSortActual
-                                { sortActualName = Id Object "Nat"
-                                , sortActualSorts = []
-                                }
-                        , SortActualSortSortActual
-                                { sortActualName = Id Object "Nat"
-                                , sortActualSorts = []
-                                }
-                        ]
-                    , sentenceSymbolReturnSort =
-                        SortActualSortSortActual
-                                { sortActualName = Id Object "Nat"
-                                , sortActualSorts = []
-                                }
-                    , sentenceSymbolAttributes =
-                        Attributes 
-                            [ MetaPattern (ApplicationPattern Application
-                                { applicationSymbolOrAlias =
-                                    SymbolOrAlias
-                                        { symbolOrAliasConstructor = Id Meta "#hook"
-                                        , symbolOrAliasParams = []
-                                        }
-                                , applicationChildren =
-                                    [ MetaPattern (StringLiteralPattern (StringLiteral "plus"))
-                                    ]
-                                })
-                            , MetaPattern (ApplicationPattern Application
-                                { applicationSymbolOrAlias =
-                                    SymbolOrAlias
-                                        { symbolOrAliasConstructor = Id Meta "#assoc"
-                                        , symbolOrAliasParams = []
-                                        }
-                                , applicationChildren = []
-                                })
-                            , MetaPattern (ApplicationPattern Application
-                                { applicationSymbolOrAlias =
-                                    SymbolOrAlias
-                                        { symbolOrAliasConstructor = Id Meta "#comm"
-                                        , symbolOrAliasParams = []
-                                        }
-                                , applicationChildren = []
-                                })
-                            ]
-                    }
-                ]
-            , moduleAttributes = Attributes []
-            }
-        ]
-    }
\ No newline at end of file
+Parse error: Error in module 'TEST-ATTRIBUTE-2' -> sort 'Nat' declaration -> attributes -> symbol or alias '#hook': Symbol '#hook' not defined..
```

Comment:
I'm not sure this test was supposed to fail... maybe the original .kore file needs to be changed? @hreada

Reply:
Fixed by not checking attribute welformednees.